plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -61,6 +63,78 @@ public void testDescribeTable()
         assertEquals(actualColumns, expectedColumns);
     }
 
+    @Test
+    public void createTableWithEveryType()
```

Comment:
Since CREATE TABLE is now supported we can add roundtrip tests for the data types to ensure our read and write mappings are in sync.

Reply:
We are using SqlExecutor for roundtrip tests, so it relates to #6748.